stylix/target.nix

```
@@ -110,7 +110,7 @@
           default = config.stylix.image != null && autoEnable;
           defaultText =
             if autoEnable then lib.literalExpression "stylix.image != null" else false;
-          example = config.stylix.image == null;
+          example = true;
```

Currently this would've evaluated to `true` when rendering docs. Was the intention:

```suggestion
          example = lib.literalExpression "stylix.image == null";
```

?

---

imo `true` would make the most sense here, telling the user to set the wallpaper enable to `stylix.image == null` is a very bad idea.

---

Actually, since we know this function (currently) never recieves a dynamic `autoEnable` value, and the new restrictive docs would catch if it ever did, we could match the `defaultText` impl and do:

```suggestion
          example =
            if autoEnable then
              lib.literalExpression "config.stylix.image == null"
            else
              false;
```

---

Done.